Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -545,7 +545,11 @@ public final class ManifestLoader: ManifestLoaderProtocol {
 
         let encoder = JSONEncoder()
         if #available(macOS 10.15, *) {
+            #if os(macOS)
             encoder.outputFormatting = [.sortedKeys, .withoutEscapingSlashes]
+            #else // `.withoutEscapingSlashes` is not in 5.3 on non-Darwin platforms
```

Comment:
Can we please move the comment `// .withoutEscapingSlashes` is not in 5.3 on non-Darwin platforms` at the `#if os(macOS)` like for `/PIFBuilder.swift` ?

Reply:
note: https://github.com/apple/swift-corelibs-foundation/pull/2889

Reply:
There doesn't seem to be a way to conditionalize on the patch component of the version, which means that this would still fail on older toolchains.  Or should this PR not be taken once the Foundation fix is in the latest 5.3 toolchains?